### ISSUE TITLE
1 day is coming less in span 

### ISSUE THREAD

**Issue by ajkosh** [philippfrenzel/yii2fullcalendar#27]
i m selecting start_date 2015-11-15 and end_date 2015-11-17.
but in calendar it is showing only 15 and 16 event span.
17 is not coming.

**Comment by philippfrenzel** [comment]
I guess it's an error in your query? Did you select the daterange like  

StartDate <= LookupDate < EndDate

**Comment by ajkosh** [comment]
i have start_date and end_date and below is my code in model that is returning object of calendar
```
public function findEvents()
    {
        $retVal=[];
        $model=self::find()->all();
            foreach ($model as $value) {
            $event = new \yii2fullcalendar\models\Event();
              $event->id = $value->id;
              $prefix=ProgramPrefix::findOne(['sub_program_type_id'=>$value->sub_program_id]);
              $maintrainer=EmpDetails::getName($value->trainer_id);
              $event->title = $prefix->prefix.' '.'['.$maintrainer.']';
              $zone=Zone::findOne([$value->zone_id]);
              $event->color=$zone->color;
              $event->allDay=true;
              $event->start = $value->start_date;
              $event->end=$value->end_date;
              $retVal[] = $event;
        }
        return $retVal;
}
```

**Comment by bhavuksuthar** [comment]
I am having the same problem. 1 less day in span.
and i am giving static dates according to the format given

**Comment by philippfrenzel** [comment]
hmm, I see that happening too... sorry for beeing late in reply, been busy;) I'll check and see what the reason can be...

**Comment by philippfrenzel** [comment]
Hi, don't you suppress sunday with the hiddenDays array?
http://fullcalendar.io/docs/display/hiddenDays/

**Comment by philippfrenzel** [comment]
Is this solved? Sorry - in #54 we need the same answer!
